src/api-client.ts

```diff
@@ -14,6 +14,9 @@ import {yubikey} from './yubikey.js'
 
 const netrc = new Netrc()
 
+export const ALLOWED_HEROKU_DOMAINS = ['heroku.com', 'herokai.com', 'herokuspace.com', 'herokudev.com'] as const
```

Comment:
Consider using Object.freeze to prevent `pop()`, `push()`, etc.

```suggestion
export const ALLOWED_HEROKU_DOMAINS = Object.freeze(['heroku.com', 'herokai.com', 'herokuspace.com', 'herokudev.com']) as const
```